Secret-agency-web/src/main/react/secret-agency/src/components/departments-page/DepartmentsPage.tsx

```diff
@@ -170,72 +179,75 @@ export class DepartmentsPage extends React.Component<{}, IDepartmentsState> {
             );
             const {city, country,latitude, longitude, specialization} = this.state.newDepartment;
             return (
-                <div className="table-wrapper">
```

Comment:
To je formátování, že to tu hlásí celý blok změněný?

Reply:
jj